hack/pre-release.sh

```diff
@@ -0,0 +1,207 @@
+#!/bin/bash
+set -euo pipefail
+# this script updates the WMCO and OCP versions in the Makefile, ssase CSV, bundle and Dockerfiles.
+
+# Include the common.sh 
+WMCO_ROOT=$(dirname "${BASH_SOURCE}")/..
+source $WMCO_ROOT/hack/common.sh
+
+CVP=false
+BASEIMAGE=false 
+# Locate and store the WMCO version from the makefile 
+# finds "WMCO_VERSION ?=" and selects text following it
+WMCO_VERSION=$(sed -n -e 's/^.*WMCO_VERSION ?= //p' Makefile)
+
+usage() {
+  echo "usage:"
+  echo "input the name of the branch you want to increment, and the x y or z value to bump"
+  echo "example: ./pre-release.sh master x release-4.15 y release-4.10 z"
```

Comment:
Do we really need this level of complexity? Why would we need to multiple releases at the same time. It is fine if we just handle `./pre-release.sh <branch> <stream>` i.e. do the updates independently. I think this will simplify some of the code below.